archetypes/uuf-component/src/main/resources/archetype-resources/src/main/component.yaml

```diff
@@ -0,0 +1,12 @@
+apis: [] # No APIs.
+
+bindings: [] # No bindings.
+
+config:
+    appName: ${componentName}
+    # User will be redirected to following URI after logging-in successfully.
```

Comment:
We don't need following config entries `loginRedirectUri`, `registerRedirectUri`, `logoutRedirectUri`